File: Python/dia_10.py
# Dia 10 = Converta a string para um número inteiro e some-a ao número inteiro declarado.

def soma_string_inteiro(numero_inteiro, string_numero):
    try:
        numero = int(string_numero)
        resultado = numero_inteiro + numero
        return resultado
    except ValueError:
        return "Erro: A string não pode ser convertida para um número inteiro."

# Número inteiro declarado
numero_inteiro = 100

# String para converter e somar
string_numero = input("Digite uma string para converter e somar ao número inteiro declarado: ")

# Chamada da função
resultado = soma_string_inteiro(numero_inteiro, string_numero)
print("Resultado da soma:", resultado)

File: JavaScript/dia_10/dia_10.js
// Dia 10 = Converta a string para um número inteiro e some-a ao número inteiro declarado.

function somaStringInteiro() {
    var numeroInteiro = parseInt(document.getElementById('numeroInteiro').value);
    var stringNumero = document.getElementById('stringNumero').value;
    
    // Verifica se a string é vazia
    if (stringNumero.trim() === '') {
        alert('Por favor, insira uma string válida.');
        return;
    }
    
    // Verifica se a string é um número válido
    var numero = parseInt(stringNumero);
    if (isNaN(numero)) {
        alert('A string não pode ser convertida para um número inteiro.');
        return;
    }
    
    var resultado = numeroInteiro + numero;
    alert('Resultado da soma: ' + resultado);
}